Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions.targets

```diff
@@ -178,4 +178,20 @@
       Condition="'$(ScriptsError)' == 'true'" />
   </Target>
 
+  <Target Name="MarkdownValidation"
```

Comment:
I get these when building OSOCE:

```
23>------ Build started: Project: Lombiq.Privacy, Configuration: Debug Any CPU ------
21>'corepack' is not recognized as an internal or external command,
21>operable program or batch file.
22>'corepack' is not recognized as an internal or external command,
22>operable program or batch file.
23>'corepack' is not recognized as an internal or external command,
23>operable program or batch file.
21>'corepack' is not recognized as an internal or external command,
21>operable program or batch file.
23>'corepack' is not recognized as an internal or external command,
23>operable program or batch file.
22>'corepack' is not recognized as an internal or external command,
22>operable program or batch file.
16>Lombiq.Hosting.Tenants.Admin.Login -> E:\ShortPath\OSOCE2\src\Modules\Lombiq.Hosting.Tenants\Lombiq.Hosting.Tenants.Admin.Login\bin\Debug\net6.0\Lombiq.Hosting.Tenants.Admin.Login.dll
17>Lombiq.Tests.UI.Shortcuts -> E:\ShortPath\OSOCE2\test\Lombiq.UITestingToolbox\Lombiq.Tests.UI.Shortcuts\bin\Debug\net6.0\Lombiq.Tests.UI.Shortcuts.dll
24>------ Build started: Project: Lombiq.Tests.UI, Configuration: Debug Any CPU ------
15>Lombiq.Hosting.Tenants.Management -> E:\ShortPath\OSOCE2\src\Modules\Lombiq.Hosting.Tenants\Lombiq.Hosting.Tenants.Management\bin\Debug\net6.0\Lombiq.Hosting.Tenants.Management.dll
18>Lombiq.Hosting.Azure.ApplicationInsights -> E:\ShortPath\OSOCE2\src\Modules\Lombiq.Hosting.Azure.ApplicationInsights\bin\Debug\net6.0\Lombiq.Hosting.Azure.ApplicationInsights.dll
19>Lombiq.HelpfulExtensions -> E:\ShortPath\OSOCE2\src\Modules\Lombiq.HelpfulExtensions\Lombiq.HelpfulExtensions\bin\Debug\net6.0\Lombiq.HelpfulExtensions.dll
23>
23>> nodejs-extensions@0.0.1 lint:markdown:args E:\ShortPath\OSOCE2\src\Modules\Lombiq.Privacy\Lombiq.Privacy\node_modules\nodejs-extensions
21>
21>> nodejs-extensions@0.0.1 lint:markdown:args E:\ShortPath\OSOCE2\src\Modules\Lombiq.UIKit\node_modules\nodejs-extensions
22>
22>> nodejs-extensions@0.0.1 lint:markdown:args E:\ShortPath\OSOCE2\src\Utilities\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.Samples\node_modules\nodejs-extensions
22>> node scripts/markdown-to-msbuild.js $npm_config_directory || AREA=markdown STEP=linting pnpm error
23>> node scripts/markdown-to-msbuild.js $npm_config_directory || AREA=markdown STEP=linting pnpm error
23>
21>> node scripts/markdown-to-msbuild.js $npm_config_directory || AREA=markdown STEP=linting pnpm error
21>
22>
23>internal/modules/cjs/loader.js:883
23>  throw err;
23>  ^
23>
23>EXEC : error : Cannot find module 'node:fs/promises'
23>Require stack:
23>- E:\ShortPath\OSOCE2\src\Modules\Lombiq.Privacy\Lombiq.Privacy\node_modules\.nx\scripts\markdown-to-msbuild.js
23>    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
23>    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
23>    at Module.require (internal/modules/cjs/loader.js:952:19)
23>    at require (internal/modules/cjs/helpers.js:88:18)
23>    at Object.<anonymous> (E:\ShortPath\OSOCE2\src\Modules\Lombiq.Privacy\Lombiq.Privacy\node_modules\.nx\scripts\markdown-to-msbuild.js:3:20)
23>    at Module._compile (internal/modules/cjs/loader.js:1063:30)
23>    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
23>    at Module.load (internal/modules/cjs/loader.js:928:32)
23>    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
23>    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12) {
23>  code: 'MODULE_NOT_FOUND',
23>  requireStack: [
23>    'E:\\ShortPath\\OSOCE2\\src\\Modules\\Lombiq.Privacy\\Lombiq.Privacy\\node_modules\\.nx\\scripts\\markdown-to-msbuild.js'
23>  ]
23>}
21>internal/modules/cjs/loader.js:883
21>  throw err;
21>  ^
21>
21>EXEC : error : Cannot find module 'node:fs/promises'
21>Require stack:
21>- E:\ShortPath\OSOCE2\src\Modules\Lombiq.UIKit\node_modules\.nx\scripts\markdown-to-msbuild.js
21>    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
21>    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
21>    at Module.require (internal/modules/cjs/loader.js:952:19)
21>    at require (internal/modules/cjs/helpers.js:88:18)
21>    at Object.<anonymous> (E:\ShortPath\OSOCE2\src\Modules\Lombiq.UIKit\node_modules\.nx\scripts\markdown-to-msbuild.js:3:20)
21>    at Module._compile (internal/modules/cjs/loader.js:1063:30)
21>    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
21>    at Module.load (internal/modules/cjs/loader.js:928:32)
21>    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
21>    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12) {
21>  code: 'MODULE_NOT_FOUND',
21>  requireStack: [
21>    'E:\\ShortPath\\OSOCE2\\src\\Modules\\Lombiq.UIKit\\node_modules\\.nx\\scripts\\markdown-to-msbuild.js'
21>  ]
21>}
22>internal/modules/cjs/loader.js:883
22>  throw err;
22>  ^
22>
22>EXEC : error : Cannot find module 'node:fs/promises'
22>Require stack:
22>- E:\ShortPath\OSOCE2\src\Utilities\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.Samples\node_modules\.nx\scripts\markdown-to-msbuild.js
22>    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
22>    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
22>    at Module.require (internal/modules/cjs/loader.js:952:19)
22>    at require (internal/modules/cjs/helpers.js:88:18)
22>    at Object.<anonymous> (E:\ShortPath\OSOCE2\src\Utilities\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.Samples\node_modules\.nx\scripts\markdown-to-msbuild.js:3:20)
22>    at Module._compile (internal/modules/cjs/loader.js:1063:30)
22>    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
22>    at Module.load (internal/modules/cjs/loader.js:928:32)
22>    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
22>    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12) {
22>  code: 'MODULE_NOT_FOUND',
22>  requireStack: [
22>    'E:\\ShortPath\\OSOCE2\\src\\Utilities\\Lombiq.NodeJs.Extensions\\Lombiq.NodeJs.Extensions.Samples\\node_modules\\.nx\\scripts\\markdown-to-msbuild.js'
22>  ]
22>}
23>
23>EXEC : > nodejs-extensions@0.0.1 error E: \ShortPath\OSOCE2\src\Modules\Lombiq.Privacy\Lombiq.Privacy\node_modules\nodejs-extensions
23>> echo "Error in $AREA pipeline: $STEP failed."
23>
21>
21>EXEC : > nodejs-extensions@0.0.1 error E: \ShortPath\OSOCE2\src\Modules\Lombiq.UIKit\node_modules\nodejs-extensions
21>> echo "Error in $AREA pipeline: $STEP failed."
22>
22>EXEC : > nodejs-extensions@0.0.1 error E: \ShortPath\OSOCE2\src\Utilities\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.Samples\node_modules\nodejs-extensions
22>> echo "Error in $AREA pipeline: $STEP failed."
22>
21>
21>Error in markdown pipeline: linting failed.
21>E:\ShortPath\OSOCE2\src\Utilities\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.targets(194,5): error MSB3073: The command "npm explore nodejs-extensions -- pnpm  lint:markdown:args --directory=." exited with code -1.
21>Done building project "Lombiq.UIKit.csproj" -- FAILED.
23>Error in markdown pipeline: linting failed.
```

Reply:
> EXEC : error : Cannot find module 'node:fs/promises'

[`node:fs/promises`](https://nodejs.org/dist/latest/docs/api/fs.html#fspromisesreadfilepath-options) is a built-in package, if Node can't find it on your machine, that sounds like a local installation problem. What version of Node do you have?

Reply:
v14.16.0, matching what we have in our guides.

Reply:
Wasn't the whole point of NodeJs-Extensions is that we can finally use v16 though?

Reply:
I've replaced it with the old style path, this should work in v14 and v16 too.

Reply:
It's the point, but until we use it everywhere, I can't upgrade because then everything else will break, nor can we use NVM to switch versions.

Reply:
What would break exactly? I have been using v16 for at least half a year and it works just fine. In fact gulp doesn't work for me on Linux at all if I use Node v14.

Reply:
Gulp doesn't support v14, let alone v16. So, I'm not venturing into trying it out.

Reply:
Some of my pain has been documented in comments under ADHOC-158, but [here's the gist](https://github.com/gulpjs/gulp/discussions/2649) of it:

> The team is working towards the 5.0 release which will be node 10+ and support node 14 & 16, etc.

Reply:
I'm happy it works for you, Dávid. Have you used any of the watch tasks while on Node.js 16, by any chance?

Reply:
Yup, works fine on v16.15.0:
![image](https://user-images.githubusercontent.com/4007293/180428056-7a5793f1-bd1a-4567-94da-dd38debde371.png)